miner/worker.go

```diff
@@ -442,12 +442,7 @@ func (miner *Miner) applyTransaction(env *environment, tx *types.Transaction) (*
 		// after the tx is deemed successful and the journal has been cleared already.
 		extraOpts = &core.ApplyTransactionOpts{
 			PostValidation: func(evm *vm.EVM, result *core.ExecutionResult) error {
```

Comment:
We should drop the post-validation diff of op-geth, and validate the access-list before execution.